assets/js/app.js

```diff
@@ -0,0 +1,59 @@
+(function() {
+var weapon1 = $('.weapon1');
+var weapon2 = $('.weapon2');
+var special = $('.special');
+
+var heroes = [Mal];
+var enemies = [allianceSoldier];
+function Game() {
+  this.on(takeTurnWithWeapon, function() {
+    enemies[1].trigger(attacked);
```

Comment:
Have the Hero and Enemy attack and trigger the `'attacked'`event from there. So it's ok to do `heroes[0].attack(enemies[0]);`

Then in `Hero.prototype.attack` call `enemy.trigger('attacked')` and pass some amount of damage as the second argument to the `trigger` function.